openmeter/ent/schema/subscription_patch.go

```diff
@@ -0,0 +1,216 @@
+package schema
```

Comment:
we discussed to apply patch at write time instead of read time and store final state in db